tests/unit/plugins/kubernetes/kubernetes_pod_executor_test.py

```diff
@@ -2,6 +2,12 @@
 from unittest import mock
 
 import pytest
+from kubernetes.client import ApiException
+from kubernetes.client import V1Container
+from kubernetes.client import V1ObjectMeta
+from kubernetes.client import V1Pod
+from kubernetes.client import V1PodSpec
+from mock import patch
```

Comment:
```suggestion
```

we probably want to use `mock` from the stdlib (e.g., what we've imported with `from unittest import mock`